iio/iio_app/iio_app.c

```diff
@@ -322,24 +316,13 @@ static int32_t irq_setup(struct no_os_irq_ctrl_desc **irq_desc)
 }
 #endif
 
-
 /**
- * @brief IIO Application API with trigger initialization.
- * @param ctx_attrs - array of context attributes.
- * @param nb_ctx_attr - number of attributes in ctx_attrs.
- * @param devices  - IIO devices to be used.
- * @param nb_devs  - Number of devices to be used.
- * @param trigs    - IIO triggers to be used.
- * @param nb_trigs - Number of triggers to be used.
- * @param irq_desc - IRQ descriptor to be used
- * @param iio_desc - IIO descriptor to be returned.
- * @return 0 in case of success or negative value otherwise.
+ * @brief Setup communication and obtain IIO descriptor.
+ *
+ * @param app - the iio application descriptor
+ * @return 0 on success, negative value otherwise
  */
-int32_t iio_app_run_with_trigs(struct iio_ctx_attr *ctx_attrs,
-			       uint32_t nb_ctx_attr,
-			       struct iio_app_device *devices, uint32_t nb_devs,
-			       struct iio_trigger_init *trigs, int32_t nb_trigs,
-			       void *irq_desc, struct iio_desc **iio_desc)
+static int iio_app_prepare(struct iio_app_desc *app)
```

Comment:
move the content of iio_app_prepare into iio_app_init

Reply:
Moved.